profiling-ffi/src/profiles/datatypes.rs

```diff
@@ -835,7 +835,6 @@ mod tests {
     #[test]
     // TODO FIX
     #[cfg_attr(miri, ignore)]
-
```

Comment:
Why the new whitespace?

Reply:
It removes the whitespace--that's the lint `clippy::empty_line_after_outer_attr`.